include/functions.us.h

```diff
@@ -4,8 +4,10 @@
 #include "common.h"
 #include "game/object.h"
 
+#ifndef SKIP_SPR_DECL
 void SprAttrSet(s16, s16, u16);
 void SprAttrReset(s16, s16, u16);
+#endif
```

Comment:
Small hack I had to do because I'm lazy. Ideally we want a proper spr header file.